tests/src/rules/no-unused-modules.js

```diff
@@ -663,6 +663,33 @@ describe('test behaviour for new file', () => {
   });
 });
 
+describe('test behaviour for destructured exports', () => {
+  before(() => {
```

Comment:
Please always use beforeEach/afterEach; before/after is “all” semantics, which leads to test pollution.